mmv1/products/privateca/CertificateTemplate.yaml

```diff
@@ -35,8 +57,241 @@ examples:
     primary_resource_id: 'default'
     vars:
       name: 'my-template'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
```

Comment:
This seems to have changed the behavior (or at least documentation) of the IAM resource based on this.

Do you know what is going on there?

https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10527-old..auto-pr-10527#diff-5dcfd60d59ef2ac2f119d71bdf3847bfc09604afa7e388e8abc050ebcc489bf0R140

Reply:
It looks like there's a bug for all mmv1 resources where the `location` field is documented as required on iam policy even though it is actually optional. I have fixed that in this PR.